common/flatpak-dir.c

```diff
@@ -1164,6 +1164,30 @@ flatpak_dir_system_helper_call (FlatpakDir   *self,
                                       error);
 }
 
+static gboolean
+flatpak_dir_system_helper_call_create_system_child_repo (FlatpakDir    *self,
```

Comment:
This function name should be consistent with the D-Bus method. Does it create a repo, or just a directory within the repo?

Reply:
This just creates a directory, yes. I should probably think of something entirely different which is more apt. Adding to my ToDo list.